package.json

```diff
@@ -17,6 +17,7 @@
     "can-stache-bindings": "^3.0.5",
     "can-types": "^1.0.0",
     "can-util": "^3.7.0",
+    "can-validate-interface": "^0.1.0",
```

Comment:
I think we can wait to make sure it works before going 1.0.  We will hide its docs.  For now, it's just a side-module used by can-connect.